primitives/consensus/common/src/lib.rs

```diff
@@ -116,16 +117,40 @@ impl RecordProof {
 	/// Returns if `Self` == `Yes`.
 	pub fn yes(&self) -> bool {
```

Comment:
This method is probably not used anywhere now. I guess every time it's `yes()` you need the `kind()` anyway, so best to match on the latter.